docs/script.rst

```diff
@@ -560,10 +560,6 @@ In order to append a string value to a file without erasing existing content, yo
 
   myFile.append('Add this line\n')
 
-Or use the `left shift` operator, a more idiomatic way to append text content to a file::
-
-  myFile << 'Add a line more\n'
-
```

Comment:
You don't need to change anything in `script.rst`. We only removed the `<<` for channels, but in this case the `<<` operator is used by files to append text to a file.

Reply:
Ok, undone.